targets/TARGET_STM/TARGET_STM32L4/device/stm32l4xx_hal_sai.c

```diff
@@ -1469,7 +1469,7 @@ void HAL_SAI_IRQHandler(SAI_HandleTypeDef *hsai)
     uint32_t cr1config = hsai->Instance->CR1;
     uint32_t tmperror;
     
-    /* SAI Fifo request interrupt occured ------------------------------------*/
+    /* SAI Fifo request interrupt occurred -----------------------------------*/
```

Comment:
for 3rd party files, we should not change them for easier integration.

@ARMmbed/team-st-mcd Please review